storm-client/src/jvm/org/apache/storm/pacemaker/PacemakerClientPool.java

```diff
@@ -54,25 +54,17 @@ public PacemakerClientPool(Map<String, Object> config) {
         }
     }
     
-    public HBMessage send(HBMessage m) throws PacemakerConnectionException {
-        try {
+    public HBMessage send(HBMessage m) throws InterruptedException {
             return getWriteClient().send(m);
-        } catch (Exception e) {
-            rotateClients();
```

Comment:
Why is it fine to remove this?

Reply:
Sorry for the late reply as I was on vacation last week.  The original code of `getWriteClient().send(m);` actually eats all the exceptions so that no exception will be thrown. I believe this `catch` never did anything. 
With this PR, only `InterruptedException` will be thrown from `getWriteClient().send(m)` and I don't think we need to `rotateClients()` when `InterruptedException` happens

Reply:
Thanks, that makes sense. rotateClients looks unused now, does it make sense to keep?

Reply:
Yea I think we can delete it. Thanks